modules/end-user-guide/pages/url-parameter-for-starting-ephemeral-workspaces.adoc

```diff
@@ -0,0 +1,30 @@
+:_content-type: CONCEPT
+:description: URL parameter starting ephemeral workspaces
+:keywords: ephemeral-workspace, persistent-workspace
+:navtitle: URL parameter starting ephemeral workspaces
+:page-aliases:
+
+[id="url-parameter-for-starting-ephemeral-workspaces_{context}"]
+= URL parameter for starting ephemeral workspaces
+
+The URL parameter for specifying a supported storage type is `storageType=__<storage_type>__`:
+
+[source,subs="+quotes,+attributes,+macros"]
+----
+pass:c,a,q[{prod-url}]#__<git_repository_url>__?storageType=__<storage_type>__
+----
+
+The URL parameter `__<storage_type>__` values for supported storage types are `ephemeral`, `per-user` and `per-workspace`.
+
+The URL for starting an ephemeral workspace:
+
+[source,subs="+quotes,+attributes,+macros"]
+----
+pass:c,a,q[{prod-url}]#__<git_repository_url>__?storageType=ephemeral
+----
+
+If the URL for starting a new workspace doesn't contain a URL parameter specifying the storage type, {prod-short} uses the default storage type defined in the `CheCluster` Custom Resource.
```

Comment:
```suggestion
If the URL for starting a new workspace does not contain a URL parameter specifying the storage type, {prod-short} uses the default storage type defined in the `CheCluster` Custom Resource.
```
will let @max-cx @themr0c review but as I recall we do not use contractions in the official docs